.github/workflows/docs-manifest.yml

```diff
@@ -0,0 +1,39 @@
+name: Keep docs manifest up to date
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches:
+      - trunk
+    paths:
+      - docs/**/*.md
+
+jobs:
+  check-docs-manifest:
+    name: Check docs manifest for staleness
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '8.4'
+        tools: composer:v2
+
+    - name: Install dependencies
+      run: composer install
```

Comment:
Can we use `--no-dev` to make it faster? We could even just run the PHP script directly to save resources; it has no dependencies.